packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.routing.js

```diff
@@ -12,6 +12,11 @@ export default /* @ngInject */ ($stateProvider) => {
       url: '?ip&dateTime',
       component: 'traffic',
       resolve: {
+        ip: /* @ngInject */ ($transition$) => $transition$.params().ip,
```

Comment:
why not using `$stateParams` instead of `$transitions` ?

Reply:
for the sake of homogeneity in the manager code.

it seems that there is more use of $transition$ than the $stateParams.

But to be honest I did not check I just applied/trust the requested changes